content/news/Showcasing/index.md

```diff
@@ -0,0 +1,32 @@
++++
+title = "Showcasing Some Final Projects from Our 2021 Session"
```

Comment:
File name (slug text) should closely mirror the title.  Here I would say:

showcasing-final-projects-2021

not "Showcasing"